### PR TITLE
Layout Editor: Automatic save

### PR DESCRIPTION
**Pull request by maurofmferrao**
relates to xibosignageltd/xibo-private#166

 - Layout background image triggers save
 - Toast messages removed
 - Loading overlay changed to a corner icon
 - Fix style for date inputs in read only mode
 - Layout Editor help added to toolbar
 - Toolbar not refreshing when reloading viewer

### CODE REVIEW

File: lib/Controller/Layout.php [hunk 1]
```diff
@@ -272,6 +272,8 @@ public function displayDesigner(Request $request, Response $response, $id)
             'users' => $this->userFactory->query(),
             'modules' => $moduleFactory->getAssignableModules(),
             'timeZones' => $timeZones,
+            'help' => $this->getHelp()->link('Layout', 'Edit'),
+
```
Comment (dasgarner) [inline lib/Controller/Layout.php:276]: phpcs

File: modules/templates/social-media-elements.xml [hunk 2]
```diff
@@ -115,7 +115,7 @@
                     <height>40</height>
                     <rotation>0</rotation>
                     <defaultProperties>
-                        <property id="fontFace">Open Sans</property>
+                        <property id="fontFace">open sans regular</property>
```
Comment (dasgarner) [inline modules/templates/social-media-elements.xml:118]: This isn't a font we ship

File: ui/src/templates/forms/region.hbs [hunk 3]
```diff
@@ -7,21 +7,21 @@
         <form id="regionFormEdit" class="XiboForm">
             <div class="tab-content">
                 <div class="tab-pane active" id="generalTab">
-                    {{> inputs/hidden id="type" value=region.type }}
-                    {{> inputs/text value=region.name id="name" title="Name" helpText="An optional name for this widget" }}
-                    {{> inputs/checkbox value=(getOption region.regionOptions "loop") id="loop" title="Loop?" helpText="If there is only one item in this region should it loop?" }}
+                    {{> inputs/hidden id="type" value=region.type customClass="position-input" }}
+                    {{> inputs/text value=region.name id="name" title="Name" helpText="An optional name for this widget" customClass="position-input" }}
```
Comment (dasgarner) [inline ui/src/templates/forms/region.hbs:11]: We have a bunch of stuff here that isn't translated, so we need to have a think about how this gets handled (not for this PR, but we can open an issue for it please)

**Review by dasgarner — CHANGES_REQUESTED** [review]
A few comments on this one (minor things)

File: lib/Controller/Layout.php (hunk 1 above)
Comment (maurofmferrao) [reply lib/Controller/Layout.php:276]: the empty new line between the new code and the end of the array right?

File: modules/templates/social-media-elements.xml (hunk 2 above)
Comment (maurofmferrao) [reply modules/templates/social-media-elements.xml]: Changed to a font in the default theme

File: ui/src/templates/forms/region.hbs (hunk 3 above)
Comment (maurofmferrao) [reply ui/src/templates/forms/region.hbs:11]: I've created a new issue for this

**Review by dasgarner — APPROVED** [review]

**Pull request merged**